internal/bus/bus.go

```diff
@@ -32,7 +32,7 @@ type Bus struct {
 
 func (b *Bus) LoadProviders() {
 	wd, _ := os.Getwd()
-	b.Manager.Autoload("agent-provider", "/system/providers", wd).Register()
+	b.Manager.Autoload("agent-provider", "/system/providers", "/usr/local/system/providers", wd).Register()
```

Comment:
Having the same structure in system makes sense to me, can I just ask, where the whole `system` comes from?

Reply:
`system` it is just a convention, it contains additional config and binaries reserved for kairos. E.g. you will find the default cloud configs of kairos inside `/system/oem`

Reply:
understood, thanks :)